packages/modal/src/components/Modal.module.css

```diff
@@ -0,0 +1,108 @@
+@keyframes fadeIn {
```

Comment:
Put local variables at top. Assign values from theme here.

Then use only local variables inside the styles.